Podfile

```diff
@@ -3,7 +3,13 @@ install! 'cocoapods',
 use_frameworks!
 
 def shared_pods
-    pod "ReactiveSwift", "~> 4.0"
+    pod "ReactiveSwift", "~> 5.0"
+end
+
+target "Example" do
+  platform :ios, "8.0"
+  pod "Kingfisher"
+  shared_pods
```

Comment:
there are some oddities here. For instance, RAC was removed from the target `Example` and everything works fine. Now the thing here is that we are linking RAC from carthage, while at the same time using CocoaPods. This will have to be fixed at a later point.